scripts/init.sh

```diff
@@ -2,10 +2,6 @@
 
 set -o nounset
 
-echo -n "This script will remove all existing iovns configurations, wallets etc. Do you want to proceed? [Y/n] "
```

Comment:
I don't think it's nice to destroy existing wallets, etc without warning.  If this is blocking some CI or script then just pipe yes into it, eg `yes | ./init.sh`.